### DISCUSSION TITLE
Green space

### DISCUSSION THREAD

**Discussion by psd** [digital-land/data-standards-backlog#64] [Planning consideration]
Green space or greenspace is an area of undeveloped land in an urban context which is not usually available for development such as playing fields, sports’ facilities, play areas and allotments.

Ordnance Survey provides an [Open Green space](https://osdatahub.os.uk/downloads/open/OpenGreenspace) product containing the boundaries with categories.

`endpoint-url` is https://api.os.uk/downloads/v1/products/OpenGreenspace/downloads?area=GB&format=GML&subformat=3&redirect

See: 
* https://en.wikipedia.org/wiki/Urban_green_space
* https://www.designingbuildings.co.uk/wiki/Green_space

**Comment by psd** [comment]
The Open Green space data contains the following features:

| Feature | Considerations | Count |
|:-|:-|-:|
| Allotments Or Community Growing Spaces | [Allotments](https://considerations.planning-data.dev/planning-consideration/allotments) | 13162 |
| Bowling Green | [Bowling greens](https://considerations.planning-data.dev/planning-consideration/bowling-greens) | 6582 |
| Cemetery | [Cemetery](https://considerations.planning-data.dev/planning-consideration/cemetery) | 7647 |
| Golf Course | [Golf courses](https://considerations.planning-data.dev/planning-consideration/golf-courses) | 2998 |
| Other Sports Facility |  | 15354 |
| Play Space | [Play spaces](https://considerations.planning-data.dev/planning-consideration/play-spaces) | 43313 |
| Playing Field | [Playing fields](https://considerations.planning-data.dev/planning-consideration/playing-fields) | 21422 |
| Public Park Or Garden | [Public parks and gardens](https://considerations.planning-data.dev/planning-consideration/public-parks-and-gardens) | 13698 |
| Religious Grounds | [Religious buildings](https://considerations.planning-data.dev/planning-consideration/religious-buildings) | 22333 |
| Tennis Court | [Tennis courts](https://considerations.planning-data.dev/planning-consideration/tennis-courts) | 6722 |

These considerations are allocated by the local planning or other local authority and categorised in different ways on different local area maps.

1. Is it helpful to the provider for us to offer individual specifications for each of these considerations?
2. Is it most useful for data consumers if we offer these as individual datasets, or should we compile these separate considerations into a single open space dataset, to match the OS Green Space dataset?

**Reply by colmjude** [reply]
I think each of these should be a separate dataset on the platform.

It follows our principle of having datasets about discrete things and often, inside LPAs, these things are treated as separate things with specific layers on their maps and specific processes for handling them.

We are going to find out if we can find specific legislation for each of these considerations.

**Reply by psd** [reply]
I concur. I think Other Sports Facility includes indoor and outdoor facilities. I think Public Park and Gardens might be a new concern.

**Reply by colmjude** [reply]
The existing dataset is split into 2 layer

* greenspace sites
* access points

### Greenspace sites

Paul has shared the split between the different functions.

Each record comes with these fields

| field | example | notes |
| --- | --- | --- |
| id | 10687F57-0A57-64A4-E063-AAEFA00A3B27 | |
| function | Play Space | this property can be used to split the dataset |
| distinctive_name_1 | St Andrew's Church | of the 153,231 features this field is blank for 109,185 |
| distinctive_name_2 | Northfields Skatepark | blank 152,909 times |
| distinctive_name_3 | Musselburgh Links | Only used 8 times |
| distinctive_name_4 | | is not used |

### Access points

There are 307,380 records in the layer.

A record consists of

| field | example |
| --- | --- |
| id | 6EBCD8C4-07C1-4D82-9E39-D0E1914F7C71 |
| access_type | Pedestrian |
| ref_to_greenspace_site | 10687FB1-665A-64A4-E063-AAEFA00A3B27 |

Break down of access_type is

| access_type | count |
| --- | --- |
| Motor Vehicle And Pedestrian | 69057 |
| Motor Vehicle | 237774 |
| Pedestrian | 549 |

**Comment by psd** [comment]
# Other sports facilities

[Indoor sports and leisure facilities](https://considerations.planning-data.dev/planning-consideration/indoor-sports-and-leisure-facilities)

These are in the OS Open Green space dataset includes indoor and outdoor council provided facilities, but contains a lot of different things such as tennis courts. There's a potential to categorise these areas which are often have a dedicated page on council websites, eg "outdoor gyms", or  "wheeled sports" ("skateparks").

**Reply by psd** [reply]
Also not all sports grounds, eg. Berkhamsted Town football club is missing:

![image](https://github.com/digital-land/data-standards-backlog/assets/5833/f7ba5b01-6ed0-408c-ad56-b88d41141eb6)

**Reply by psd** [reply]
Sport England provide [open data](https://www.activeplacespower.com/pages/downloads) on the location and types of sports facilities we could match to these entities and use or link to their categories.

**Comment by kieranwint** [comment]
@clairemiller2 recently published a blog post on [our approach to open space](https://digital-land.github.io/blog-post/our-approach-to-open-space/), bringing to light the range in clarity on definitions for open space designations. 

As a result of some research on this subject (using the NPPF, planning practice guidance, and LPA policy documentation), when it comes to the less clearly defined local open space typologies, we propose, in the long term, asking for LPAs to provide their data using the following open space typologies:

- Amenity Greenspace - Informal recreation spaces, communal green spaces in and around housing, village greens, urban commons, and civic spaces
- Public Parks and Gardens - Accessible, high quality opportunities for informal recreation and community events, including urban parks, country parks, regional parks, forest parks, and formal gardens
- Natural and Semi-Natural Greenspace - Informal and natural green space providing the opportunity to promote meaningful and safe recreation, including woodland and scrub, grassland, heath or moor, wetlands, open and running water, and wastelands
- Children’s play space - Designated areas for children and young people containing a range of facilities and an environment that has been designed to provide focused opportunities for outdoor play, including Local Areas for Play (LAPs), Local Equipped Area for Play (LEAPs), Neighbourhood Equipped Area for Play (NEAPs) and Multi-use Games Area (MUGAs)
- Allotments - Allotments, community gardens and community farms
- Cemeteries - Includes churchyards and other burial grounds. Areas for quiet contemplation and burial of the dead, but also often with high biodiversity value
- Playing pitches - Pitch sports including football, rugby union, rugby league, hockey, lacrosse, cricket, American football etc
- Other outdoor sports - Courts and greens comprising natural or artificial surfaces, including tennis courts, bowling greens, athletics tracks and other outdoor sports areas

It is worth noting that capturing the majority of these local open space typologies (5/8) will be achieved through the use of the OS Open Greenspace dataset, but as mentioned, our long term aspiration would be for LPAs to provide this data.

If you have any thoughts on the proposed approach, please let us know! Equally, please do have a read of the blog post for more detail!

**Comment by drewhardy77** [comment]
Some thoughts on allotments specifically. Allotment sites owned by local authorities can be designated as 'statutory' or 'temporary'. 'Statutory' sites have their own protection under the Allotments Act 1925. This designation is provided by the LPA. Therefore from a planning protection point of view, we need to understand the designation of allotments by the LPA, not just where they are situated.
